charts/ingress-nginx/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: ingress-nginx
 # When the version is modified, make sure the artifacthub.io/changes list is updated
 # Also update CHANGELOG.md
-version: 3.31.0
+version: 3.32.0
 appVersion: 0.46.0
 home: https://github.com/kubernetes/ingress-nginx
 description: Ingress controller for Kubernetes using NGINX as a reverse proxy and load balancer
```

Comment:
Can you please update the changes and the CHANGELOG.md as stated in the comments?

Thanks!

Reply:
I had added them and then I was asked to delete them. Apologize and thank you for the review.